### ISSUE TITLE
ExecutiveTests won't run

### ISSUE THREAD

**Issue by dougajmcdonald** [gareththegeek/corewar#7]
The ExecutiveTests.ts file has a custom matcher which wasn't playing nicely with typescript. It should work but is giving errors running the tests.

```
declare namespace jasmine {
    interface Matchers<T> {
        toEqualInstruction(expected: IInstruction): boolean;
    }
}
```

The code above should work, but I'm still seeing errors.

I've renamed the file so it's not picked up by the Jasmine test runner for the time being

**Comment by dougajmcdonald** [comment]
I can get round this by amending the .d.ts for jasmine directly but for some reason jasmine-ts doesn't pick up the namespace extension

**Comment by dougajmcdonald** [comment]
Logged issue with `jasmine-ts` to see if they can shed any light
https://github.com/svi3c/jasmine-ts/issues/17

**Comment by dougajmcdonald** [comment]
Fixed with https://github.com/gareththegeek/corewar/pull/13

**Comment by dougajmcdonald** [comment]
Fixed with #36 via the response to this issue raised with the `ts-node` guys.

https://github.com/TypeStrong/ts-node/issues/462
